### DISCUSSION TITLE
How are you supposed to upload a document for GitHub education verification if you're an online student?

### DISCUSSION THREAD

**Discussion by braden8706** [community/community#140336] [GitHub Education]
### Hi everyone,

Uploading proof of being a student is nearly impossible if you're an online student. Why are you able to only take a picture of a document instead of just uploading the document itself? That's a very poor design for the application process. 
The instructions say you can upload a document of proof with the letterhead but there's absolutely no way to upload the document.
![image](https://github.com/user-attachments/assets/4c4531b7-dade-4613-b720-8f8b65cbee24)

You can't even contact someone to review it.

What do you do?

**Comment by Mti-isf** [comment]
kook143125@gmail.com

**Reply by braden8706** [reply]
Email address with no context? No thank you

**Comment by AESMatias** [comment]
Hello @braden8706! I've faced the same issue, why don't you try by taking a screenshot of the document provided by your university/institution?  It worked for me before.
Nevertheless, if this is not possible for some reason, try by submitting a ticket on https://support.github.com/request explaining your situation, i hope this help you!

**Reply by braden8706** [reply]
See I would, it'd make total sense but there's no way to upload pictures. It only gives me the option to take the picture using my webcam.
![image](https://github.com/user-attachments/assets/cd1ae5e1-6d46-408b-9482-1605d271e679)

**Reply by AESMatias** [reply]
I see, you can take the picture with your phone. It's not the best thing to do so, i agree with you on that, but i think it will work.
In my case, i don't remember "be forced" to take a picture in that way, but i'm not sure. A lot of students are facing problems with the submit platform lately though.
